app-i18n/mozc/mozc-2.20.2677.102.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="Japanese input method editor"
+DESCRIPTION="A Japanese Input Method Editor (IME) designed for multi-platform. \
+This is open source version."
+HOMEPAGE="https://github.com/google/mozc/"
+COPYRIGHT="2010-2016, Google Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+fileVersion="v2.20.2677.102.3"
+SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
+CHECKSUM_SHA256="5b7c8c1e114b99221a2d9cf650fadd1cad9137986c94762f490a62af1048a8b1"
+SOURCE_DIR="mozc"
+if [ $effectiveTargetArchitecture == "x86" ]; then
```

Comment:
Not required for x86_64. Mozc defines own int32 and uint32 which conflicts with defiend by Haiku on x86. But not on x86_64.